package/MDAnalysis/converters/RDKitParser.py

```diff
@@ -156,6 +153,11 @@ class RDKitParser(TopologyReaderBase):
     .. versionadded:: 2.0.0
     .. versionchanged:: 2.1.0
        Added R/S chirality support
+    .. versionchanged:: 2.7.0
+       Removed type guessing (attributes guessing takes place now
+       through universe.guess_TopologyAttributes() API). If atoms types is not
+       provided, type attributed get the same values as the element attribute.
```

Comment:
```suggestion
       provided, the type attribute get the same values as the element attribute.
```

Just a typo and suggestion!